### DISCUSSION TITLE
Поиск в Видимости полей

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#496] [Есть идея, а что если...]
Добрый день,
Мы периодически сталкиваемся с маленьким неудобством, решил его тут озвучить. Когда много полей скрыто и нужно какое-то определенное показать - удобно было бы иметь поиск для этого. Как считаете?

![Screenshot_2](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/851ad2aa-8571-4591-85fb-41db851f5712)

**Comment by ttm-alexeypolunin** [comment]
Да, надо сделать
